modules/planning/conf/planner_open_space_config.pb.txt

```diff
@@ -9,7 +9,7 @@ warm_start_config {
   next_node_num: 10
   step_size: 0.5
   traj_forward_penalty: 1.0
-  traj_back_penalty: 1.0
+  traj_back_penalty: 1.5
```

Comment:
Change it on purpose?

Reply:
Yes, but I will put it into a separate commit later with other changes to help stabilize Hybrid Astar in manual mode.